v3/src/components/slider/editable-slider-value.tsx

```diff
@@ -29,6 +30,15 @@
     setCandidate(value)
   }
 
+  const handleSubmitValue = (e: React.FocusEvent<HTMLInputElement>) => {
+    const inputValue = parseFloat(e.target.value)
+    if (!isNaN(inputValue)) {
+      if (inputValue < sliderModel.axis.min) sliderModel.setAxisMin(inputValue - kAxisGap)
+      if (inputValue > sliderModel.axis.max) sliderModel.setAxisMax(inputValue + kAxisGap)
```

Comment:
Rather than kAxisGap, I suggest using the method in V2's SliderModel:encompassValue where the gap is dynamically computed to 1/10 the slider's range. That's at slider_model.js line 149.